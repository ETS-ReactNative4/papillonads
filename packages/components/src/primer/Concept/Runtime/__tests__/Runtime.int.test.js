import {
  renderSpecificationsStandardsGuidelines,
  renderOpenSourceDependencyProjects,
  renderDesignSystemCore,
  renderProductMicrosites,
} from './Runtime.int.render'

describe('<Runtime />', () => {
  describe('Render', () => {
    test('must match renderSpecificationsStandardsGuidelines()', () => {
      expect(global.renderToJSON(renderSpecificationsStandardsGuidelines())).toMatchSnapshot()
    })
    test('must match renderOpenSourceDependencyProjects()', () => {
      expect(global.renderToJSON(renderOpenSourceDependencyProjects())).toMatchSnapshot()
    })
    test('must match renderDesignSystemCore()', () => {
      expect(global.renderToJSON(renderDesignSystemCore())).toMatchSnapshot()
    })
    test('must match renderProductMicrosites()', () => {
      expect(global.renderToJSON(renderProductMicrosites())).toMatchSnapshot()
    })
  })
})
