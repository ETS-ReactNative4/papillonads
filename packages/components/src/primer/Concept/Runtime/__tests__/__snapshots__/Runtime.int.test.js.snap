// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Runtime /> Render must match designSystemCore() 1`] = `
<div>
  <div>
    <h2>
      📗 Papillon Ads Implementation
    </h2>
    <h3>
      📖. Design System Core
    </h3>
    <h4>
      📃. Setup file(s)
    </h4>
    <div>
      ﹘ Coming soon.
    </div>
    <h4>
      📃. Knowledge base
    </h4>
    <div>
      ﹘ Coming soon.
    </div>
  </div>
</div>
`;

exports[`<Runtime /> Render must match openSourceDependencyProjects() 1`] = `
<div>
  <div>
    <h2>
      📗 Open Source Dependency Projects
    </h2>
    <br />
    <br />
    <h3>
      📖. Client
    </h3>
    <br />
    <h4>
      📃. Google Chrome
    </h4>
    <div>
      ﹘ 
      <a
        href="https://github.com/googlechrome"
      >
        GitHub Organization
      </a>
    </div>
    <h4>
      📃. Mozilla Firefox
    </h4>
    <div>
      ﹘ 
      <a
        href="https://github.com/mozilla"
      >
        GitHub Organization
      </a>
    </div>
    <h4>
      📃. Apple Safari
    </h4>
    <div>
      📝. Coming soon 🐬
    </div>
    <h4>
      📃. Microsoft Edge
    </h4>
    <div>
      ﹘ 
      <a
        href="https://github.com/MicrosoftEdge"
      >
        GitHub Organization
      </a>
    </div>
    <h4>
      📃. Microsoft IE11
    </h4>
    <div>
      📝. Coming soon 🐬
    </div>
    <br />
    <br />
    <h3>
      📖. Server
    </h3>
    <br />
    <h4>
      📃. Node.JS
    </h4>
    <div>
      ﹘ 
      <a
        href="https://github.com/nodejs"
      >
        GitHub Organization
      </a>
    </div>
  </div>
</div>
`;

exports[`<Runtime /> Render must match productMicrosites() 1`] = `
<div>
  <div>
    <h2>
      📗 Papillon Ads Implementation
    </h2>
    <h3>
      📖. Product Microsites
    </h3>
    <h4>
      📃. Setup file(s)
    </h4>
    <div>
      ﹘ Coming soon.
    </div>
    <h4>
      📃. Knowledge base
    </h4>
    <div>
      ﹘ Coming soon.
    </div>
  </div>
</div>
`;

exports[`<Runtime /> Render must match specificationsStandardsGuidelines() 1`] = `
<div>
  <div>
    <h2>
      📗 Specifications/Standards/Guidelines
    </h2>
    <br />
    <br />
    <h3>
      📖. Client
    </h3>
    <br />
    <h4>
      📃. Google Chrome
    </h4>
    <div>
      ﹘ 
      <a
        href="https://developer.chrome.com/home"
      >
        Google Documentation
      </a>
    </div>
    <div>
      ﹘ 
      <a
        href="https://www.google.com/chrome/"
      >
        Download
      </a>
    </div>
    <h4>
      📃. Mozilla Firefox
    </h4>
    <div>
      ﹘ 
      <a
        href="https://developer.mozilla.org/en-US/docs/Mozilla/Firefox"
      >
        Mozilla Documentation
      </a>
    </div>
    <div>
      ﹘ 
      <a
        href="https://www.mozilla.org/en-US/firefox/new/"
      >
        Download
      </a>
    </div>
    <h4>
      📃. Apple Safari
    </h4>
    <div>
      ﹘ 
      <a
        href="https://developer.apple.com/library/archive/navigation/index.html?filter=safari"
      >
        Apple Documentation
      </a>
    </div>
    <div>
      ﹘ 
      <a
        href="https://www.apple.com/safari/"
      >
        Download
      </a>
    </div>
    <h4>
      📃. Microsoft Edge
    </h4>
    <div>
      ﹘ 
      <a
        href="https://docs.microsoft.com/en-us/microsoft-edge/"
      >
        Microsoft Documentation
      </a>
    </div>
    <div>
      ﹘ 
      <a
        href="https://www.microsoft.com/en-us/edge/"
      >
        Download
      </a>
    </div>
    <h4>
      📃. Microsoft IE11
    </h4>
    <div>
      ﹘ 
      <a
        href="https://docs.microsoft.com/en-us/internet-explorer/"
      >
        Microsoft Documentation
      </a>
    </div>
    <div>
      ﹘ 
      <a
        href="https://support.microsoft.com/en-us/help/18520/download-internet-explorer-11-offline-installer"
      >
        Download
      </a>
    </div>
    <br />
    <br />
    <h3>
      📖. Server
    </h3>
    <br />
    <h4>
      📃. Node.JS
    </h4>
    <div>
      ﹘ 
      <a
        href="https://nodejs.org/en/docs/"
      >
        Node.JS Documentation
      </a>
    </div>
    <div>
      ﹘ 
      <a
        href="https://nodejs.org/en/"
      >
        Download
      </a>
    </div>
  </div>
</div>
`;
